buildpacks/nodejs-function-invoker/fixtures/package-json-no-main/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "main": "index.js"
```

Comment:
This fixture is named `package-json-no-main` which made me think it would contain a `package.json` that didn't have a `main` key, but it's actually that the `main` file is missing. Do we need two fixtures here - one with a missing main key named perhaps `package-json-no-main`, and a second with `index.js` missing (like this fixture) named something else?

Also, I realise now I missed out the `main` entry (and index.js) from my fixtures added in #47 - would you mind adding it to them now? These could be used to test the "working" case of `run_initial_checks`.

Reply:
Agreed, I can make `package-json-no-main` and `package-json-no-main-file` or something? I'll add to the fixtures too.